### ISSUE TITLE
serving/building static content

### ISSUE THREAD

**Issue by PavelPolyakov** [jxnblk/mdx-deck#20]
🤚,

Wanted to show an image in my presentation. So I put it near the `.mdx` file and addressed in the presentation as `![](flow.png)`, which was resolved into `<img class="sc-kpOJdX eTwYLo" src="./flow.png">`.

All good, but the image was not served via dev server.
Neither the image was added to the `dist` folder during the `build`.

I think all the files relative to the `.mdx` file which is being built/served - should be available via `http://`.

Regards,

**Comment by jxnblk** [comment]
Thanks! This should be possible with https://github.com/koajs/static – I opted to keep the dev server as light as possible to start with. For now, I'd recommend using a CDN – which will continue to work if you decide to deploy the exported bundle

**Comment by PavelPolyakov** [comment]
@jxnblk thanks for the response.

What do you mean by CDN? if you mean this from the deploy perspective - yes, I understand that if I put the static files to the folder and deploy everything it will work :)

But from the dev server perspective - are you going to include that middleware? Or it's already possible to include it - how then?

**Comment by jxnblk** [comment]
Sorry if that was unclear; I was referring to hosting the image assets on a CDN – Zeit's Now is a quick way to do that: https://zeit.co/now

**Comment by PavelPolyakov** [comment]
@jxnblk yep, this will help. But still I think it'll be good to have this by default.

Thanks for your work and this project!
